src/pages/login.tsx

```diff
@@ -11,33 +11,32 @@ const loginSchema = yup.object().shape({
 })
 
 function LoginForm() {
-  const [clicked, setClicked] = React.useState(false)
   const [isSubmitted, setIsSubmitted] = React.useState(false)
+  const [isError, setIsError] = React.useState(false)
```

Comment:
I fixed this on Epic React today and migrated it over so it just happened to be in this same PR.